src/components/_/TipTap.vue

```diff
@@ -172,7 +172,7 @@
 <script setup lang="ts">
 import { useEditor, EditorContent } from '@tiptap/vue-3'
 import { Link } from '@tiptap/extension-link'
-import TextAlign from '@tiptap/extension-text-align'
+import { TextAlign } from '@tiptap/extension-text-align'
```

Comment:
cc @myyxl always a good idea to import only certain things from a package, not everything (but for this package I suspect that there's not much more in it, but I didn't check)